fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -55,6 +55,12 @@ def self.cruise_lane(platform, lane, parameters = nil, env = nil)
       e = nil
       begin
         ff.runner.execute(lane, platform, parameters)
+      rescue NameError => ex
+        print_lane_context
+        line = ex.backtrace.first.match("Fastfile:(\\d+):")[1]
```

Comment:
I ran this with a good `Fastfile` and a `Snapfile` with the contents of...
```rb
Upload
```
and I got an error on this line 🤔 

```sh
bundler: failed to load command: fastlane (/Users/josh/.gem/ruby/2.3.6/bin/fastlane)
NoMethodError: [!] undefined method `[]' for nil:NilClass
  /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/lane_manager.rb:60:in `rescue in cruise_lane'
  /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
  /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
  /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
```

Any ideas? 😇